src/components/tables/cells/stats/Header.tsx

```diff
@@ -37,32 +46,61 @@ const StatsHeader = ({ header, selectableTableStoreName }: Props) => {
         SelectableTableStore['query']['count']
     >(selectableTableStoreName, selectableTableStoreSelectors.query.count);
 
+    const [firstColHeader, secondColHeader] = useMemo(() => {
+        return [
+            intl.formatMessage(
+                {
+                    id: headerSuffix ? headerSuffix : 'data.written',
+                },
+                {
+                    type: intl.formatMessage({
+                        id: header ?? 'data.data',
+                    }),
+                }
+            ),
+            intl.formatMessage(
+                {
+                    id: headerSuffix ? headerSuffix : 'data.read',
+                },
+                {
+                    type: intl.formatMessage({
+                        id: header ?? 'data.docs',
+                    }),
+                }
+            ),
+        ];
+    }, [header, headerSuffix, intl]);
+
     return (
         <>
             <TableCell>
                 <Typography
                     component="span"
                     sx={{ mt: 0.5, fontWeight: 500, whiteSpace: 'nowrap' }}
                 >
-                    <FormattedMessage
-                        id="data.written"
-                        values={{
-                            type: <FormattedMessage id={header} />,
-                        }}
-                    />
+                    {firstColHeader}
                 </Typography>
             </TableCell>
             <TableCell>
-                <DateFilter
-                    header={header}
-                    disabled={
-                        !hasStats ||
-                        isValidating ||
-                        networkFailed ||
-                        queryCount === 0
-                    }
-                    selectableTableStoreName={selectableTableStoreName}
-                />
+                {hideFilter ? (
+                    <Typography
+                        component="span"
+                        sx={{ mt: 0.5, fontWeight: 500, whiteSpace: 'nowrap' }}
+                    >
+                        {secondColHeader}
+                    </Typography>
+                ) : (
```

Comment:
Biggest changed so that we can use the stats columns in collections for the part of the columns that do not need the filter